### ISSUE TITLE
官方建议新用户使用`--mount` 代替 `-v`  Docker 17.06+

### ISSUE THREAD

**Issue by khs1994** [yeasy/docker_practice#214]
### your OS

* [ ] Linux

   * [ ] CentOS 7

   * [ ] Ubuntu 14.04

   * [ ] Ubuntu 16.04

   * [ ] Ubuntu 17.04

   * [ ] Ubuntu 17.10

   * [ ] CoreOS

   * [ ] Debian 7.7

   * [ ] Debian 8

   * [ ] Debian 9

* [x] macOS

* [ ] Windows 10

### Docker CE Version

* [x] 17.10 edge

* [ ] 17.09 stable

* [ ] 1.13.0

* [ ] 1.8.0

### Problem description

https://docs.docker.com/engine/admin/volumes/volumes/

官方建议新用户使用`--mount` 代替 `-v`

**Comment by yeasy** [comment]
That's true:
```
Tip: New users should use the --mount syntax. Experienced users may be more familiar with the -v or --volume syntax, but are encouraged to use --mount, because research has shown it to be easier to use.
```

Feel free to submit a pr to include the note. And pls include the "Fix issue#214" in the commit msg.

**Comment by khs1994** [comment]
这几天正在仔细学习官方文档，先提问题，过几天好好修改内容。
